src/compiler/generator.js

```diff
@@ -77,6 +77,22 @@ const generator = (node) => {
         value: `${stylePrefix}${node.props.bullet} ${renderedChildren.value}\n`,
         styles: renderedChildren.styles
       };
+    case 'img':
+      const { src, width, height, style } = node.props
```

Comment:
This is odd, but if you remove this destructuring and use the variables straight from `node.props`, the old tests  start working again. Please try it and tell me how it went, but old tests shouldn't break.

Reply:
Solved making `const` _block-scoped_. Seems that the tests doesn't take the right `style` prop, since were duplicated.